subworkflows/local/hicpro.nf

```diff
@@ -0,0 +1,132 @@
+/*
+ * HICPRO
+ * MAIN WORKFLOW
+ * From the raw sequencing reads to the list of valid interactions
+ */
+  
+include { HICPRO_MAPPING } from './hicpro_mapping'
+include { GET_VALID_INTERACTION } from '../../modules/local/hicpro/get_valid_interaction'
+include { GET_VALID_INTERACTION_DNASE } from '../../modules/local/hicpro/get_valid_interaction_dnase'
+include { MERGE_VALID_INTERACTION } from '../../modules/local/hicpro/merge_valid_interaction'
+include { MERGE_STATS } from '../../modules/local/hicpro/merge_stats'
+include { HICPRO2PAIRS } from '../../modules/local/hicpro/hicpro2pairs'
+include { BUILD_CONTACT_MAPS } from '../../modules/local/hicpro/build_contact_maps'
+include { ICE_NORMALIZATION } from '../../modules/local/hicpro/run_ice'
+
+// Remove meta.chunks
+def removeChunks(row){
+  meta = row[0].clone()
+  meta.remove('chunk')
+  return [meta, row[1]]
+}
+
+workflow HICPRO {
+
+  take:
+  reads // [meta, read1, read2]
+  index // path
+  fragments // path
+  chrsize // path
+  ligation_site // value
+  map_res // values
+
+  main:
+  ch_versions = Channel.empty()
+
+  // Fastq to paired-end bam
+  HICPRO_MAPPING(
+    reads,
+    index,
+    ligation_site
+  )
+  ch_versions = ch_versions.mix(HICPRO_MAPPING.out.versions)
+
+  //***************************************
+  // DIGESTION PROTOCOLS
+
+  if (!params.dnase){
+    GET_VALID_INTERACTION (
```

Comment:
```suggestion
    GET_VALID_INTERACTION (
```
Do you prefer a space or no space? `GET_VALID_INTERACTION (` or `GET_VALID_INTERACTION(`

Reply:
I'm waiting for the nextflow linter to correct (or not) people on that

Reply:
Soon:tm:

Reply:
I'm soooo waiting for this.
My latest refactoring PR, is full of `code polishing` for syntax like that...